### PR TITLE
Move 3.5.2 to 3.1.3 and modify to resolve #1522

### LINKED ISSUE #1522: 3.5.2 doesn't seem like it is in context
3.5.2 sounds more like it should either:

1. Be in section [2.10](https://github.com/OWASP/ASVS/blob/master/5.0/en/0x11-V2-Authentication.md#v210-service-authentication) where we discuss intra-service authentication 
2. It should be made more generic to be clear that this is a general guideline that temporary session tokens should be used after authentication and never a static identifier. 

| # | Description | L1 | L2 | L3 | CWE | [NIST &sect;](https://pages.nist.gov/800-63-3/sp800-63b.html) |
| :---: | :--- | :---: | :---: | :---: | :---: | :---: |
| **[3.5.2](https://appsecg.host/3.5)** | [GRAMMAR] Verify that the application uses session tokens rather than static API secrets and keys, except with legacy implementations. | | ✓ | ✓ | 798 | |

History:

| # | Description | L1 | L2 | L3 | CWE | [NIST &sect;](https://pages.nist.gov/800-63-3/sp800-63b.html) |
| :---: | :--- | :---: | :---: | :---: | :---: | :---: |
| **[3.5.2](https://github.com/OWASP/ASVS/blame/af06e16ed3612ed0da80b677bf3be04234a1d31c/4.0/en/0x12-V3-Session-management.md#L62)** | Verify the application uses session tokens rather than static API secrets and keys, except with legacy implementations. |  | ✓ | ✓ | - | 798 | tbd |
| **[3.5.2](https://github.com/OWASP/ASVS/blame/54629299a244ab712d44cc440800dbd7e8141f00/4.0/en/0x12-V3-Session-management.md#L61)** | Verify that single factor, static API secrets and keys are not used, except with legacy implementations. |  | ✓ | ✓ | - | 798 | tbd |
| **[3.3.1](https://github.com/OWASP/ASVS/blame/c0f9c295ea57ecfba848cfbdca850034825117f1/4.0/en/0x12-V3-Session-management.md)** | Verify single factor unchanging API secrets and keys are not used, except with legacy implementations. |  | ✓ | ✓ | 1.0 |

### PR DESCRIPTION
**Pull request by tghosth**
This Pull Request relates to issue #1522

### CODE REVIEW

File: 5.0/en/0x12-V3-Session-management.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ As previously noted, these requirements have been adapted to be a compliant subs
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
 | **3.1.1** | [DELETED, MERGED TO 8.3.1] | | | | | |
 | **3.1.2** | [ADDED] Verify that the application performs all session token verification using a trusted, back-end service. | ✓ | ✓ | ✓ | 603 | |
+| **3.1.3** | [MOVED FROM 3.5.2, MODIFIED] Verify that the application uses either **cryptographically signed or opaque tokens** for session management. Static API secrets and keys should be avoided. | ✓ | ✓ | ✓ | 798 | 7.1 |
```
Comment (Sjord) [inline 5.0/en/0x12-V3-Session-management.md:20]: > `**cryptographically signed or opaque tokens**`

I don't think it is a good idea to use markdown in the table. It messes with CSV export, for example. Also, I don't think the emphasis adds anything here.

Comment (tghosth) [reply 5.0/en/0x12-V3-Session-management.md]: Great point @Sjord! I have removed that

**Comment by elarlang** [comment]
3.5.2 is level 2
3.1.3 is level 1

is the change intended?

**Comment by tghosth** [comment]
Yes since this is now a more general/basic requirement it should be level 1 @elarlang

**Review by elarlang — APPROVED** [review]

**Pull request merged**